internal/host/monitor.go

```diff
@@ -162,33 +162,45 @@ func (m *Manager) clusterHostMonitoring() int64 {
 			break
 		}
 
+		log.Debug("Started cluster host monitoring cycle")
```

Comment:
Will these debug messages be left in the code?

Reply:
Yes